app/views/teacher_interface/application_forms/show/_declined.html.erb

```diff
@@ -1,8 +1,10 @@
+<h1>Apply for qualified teacher status (QTS)</h2>
```

Comment:
I think this will already be visible from: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/ec537b9518625b0d3039715936d0e6ba65988e3c/app/views/teacher_interface/application_forms/show.html.erb#L3